config/core/test-configs.yaml

```diff
@@ -2842,6 +2842,11 @@ test_configs:
   - device_type: sun50i-h5-libretech-all-h3-cc
     test_plans:
       - baseline
+      - baseline-nfs
+      - kselftest-alsa
```

Comment:
Passing in staging: https://staging.kernelci.org/test/plan/id/62a99cc604f87fd98974f1e6/ (needs some defconfig updates to be a bit more useful, they're in progress upstream.)